cdap-ui/package.json

```diff
@@ -146,95 +146,97 @@
   "dependencies": {
     "@apollo/react-hooks": "3.1.3",
     "@babel/polyfill": "7.0.0",
-    "@material-ui/core": "4.0.1",
-    "@material-ui/icons": "4.0.1",
-    "ace-builds": "1.4.4",
-    "apollo-boost": "0.4.4",
-    "apollo-server": "2.6.7",
-    "apollo-server-express": "2.6.7",
-    "body-parser": "1.14.2",
-    "bootstrap": "4.1.3",
+    "@material-ui/core": "4.8.2",
+    "@material-ui/icons": "4.5.1",
+    "ace-builds": "1.4.7",
+    "apollo-boost": "0.4.7",
+    "apollo-server": "2.9.15",
+    "apollo-server-express": "2.9.15",
+    "body-parser": "1.19.0",
+    "bootstrap": "4.4.1",
     "cask-datavoyager": "2.0.0-alpha.12.9",
     "cdap-avsc": "4.1.13",
-    "classnames": "2.2.5",
-    "clipboard": "1.7.1",
-    "compression": "1.6.1",
-    "cookie-parser": "1.4.1",
-    "css-vars-ponyfill": "1.8.0",
-    "d3": "4.12.2",
-    "dataloader": "1.4.0",
-    "ejs": "2.6.1",
-    "event-emitter": "0.3.4",
-    "express": "4.16.2",
-    "express-http-proxy": "1.5.1",
-    "finalhandler": "0.4.1",
-    "font-awesome": "4.6.3",
+    "classnames": "2.2.6",
+    "clipboard": "2.0.4",
+    "compression": "1.7.4",
+    "cookie-parser": "1.4.4",
+    "css-vars-ponyfill": "2.1.2",
+    "d3": "5.15.0",
+    "dataloader": "2.0.0",
+    "ejs": "3.0.1",
+    "event-emitter": "0.3.5",
+    "express": "4.17.1",
+    "express-http-proxy": "1.6.0",
+    "finalhandler": "1.1.2",
+    "font-awesome": "4.7.0",
     "frameguard": "3.1.0",
     "fuse.js": "3.4.6",
-    "graphql": "14.4.0",
+    "global": "^4.4.0",
+    "graphql": "14.5.8",
     "graphql-import": "0.7.1",
-    "helmet-csp": "2.7.1",
-    "history": "4.9.0",
+    "helmet-csp": "2.9.4",
+    "history": "4.10.1",
     "hsts": "2.2.0",
-    "i18n-react": "0.3.0",
-    "immutability-helper": "2.7.1",
+    "i18n-react": "0.7.0",
+    "immutability-helper": "3.0.1",
     "immutable": "3.8.2",
-    "intersection-observer": "0.4.2",
+    "intersection-observer": "0.7.0",
     "ip": "1.1.5",
-    "jexl": "2.1.1",
-    "js-file-download": "0.4.1",
+    "jexl": "2.2.2",
+    "js-file-download": "0.4.9",
     "jsplumb": "2.5.12",
-    "lodash": "4.17.10",
-    "log4js": "4.5.1",
-    "marksy": "7.0.1",
-    "modclean": "2.1.2",
-    "moment": "2.21.0",
-    "moment-timezone": "0.5.17",
-    "mousetrap": "1.6.0",
-    "ngreact": "0.3.0",
-    "numeral": "1.5.3",
-    "object-hash": "1.1.0",
-    "prismjs": "1.16.0",
-    "prop-types": "15.5.10",
-    "q": "1.4.1",
-    "query-string": "4.3.2",
-    "react": "16.8.6",
+    "lodash": "4.17.15",
+    "log4js": "6.1.0",
+    "marksy": "8.0.0",
+    "modclean": "3.0.0-beta.1",
```

Comment:
Can we avoid using beta versions? If there is a 2.9 or latest stable would be ok. We can bump to major version once it is out of beta.

Reply:
The last version released before this one is 2.1.2 (which we had before), so I downgraded it back to 2.1.2.